wallet/i18n/de.json

```diff
@@ -286,24 +291,56 @@
   "get_balance_failed": "Gleichgewicht abrufen fehlgeschlagen",
   "no_data_saved": "Keine Daten gespeichert",
   "video_player_network_error": "Das Video kann momentan nicht abgespielt werden. Bitte überprüfen Sie Ihre Internetverbindung und versuchen Sie es erneut.",
-  "something_wrong_signing_transaction": "Beim Signieren der Transaktion ist etwas schief gelaufen",
+  "something_wrong_signing_transaction": "Beim Signieren der Transaktion ist etwas schief gelaufen. Bitte versuche es erneut.",
   "no_profile_found": "Kein Profil im Geldbörse gefunden",
+  "insufficient_balance_msg": "Sie haben nicht genug Pylon, um diese Transaktion abzuschließen. Bitte klicken Sie unten, um Pylon hinzuzufügen, und versuchen Sie es erneut.",
+  "insufficient_balance": "Mangelhaftes Gleichgewicht",
+  "add_pylons": "Pylon hinzufügen",
+  "bonus_fifty": "(+50 bonus)",
+  "bonus_hundred": "(+100 bonus)",
+  "add_fund": "Fonds hinzufügen",
+  "done": "Fertig",
+  "duration": "Dauer",
+  "bonus_five": "(+5 Bonus)",
+  "bonus_ten": "(+10 Bonus)",
+  "no_internet_connection": "Bitte überprüfen Sie Ihre Internetverbindung und versuchen Sie es erneut. Wenn dieses Problem weiterhin besteht, senden Sie uns bitte eine E-Mail: ",
   "purchase_successful": "Kauf erfolgreich",
-  "done": "gedaan",
-  "add_pylons" : "Pylon hinzufügen",
-  "update_now" : "Update nu",
-  "skip": "Überspringen",
   "view": "visie",
-  "views": "keer bekeken",
+  "views": "Ansichten",
+  "jan": "Jän",
+  "feb": "Feb",
+  "mar": "März",
+  "apr": "Apr",
+  "may": "Mai",
+  "jun": "Juni",
+  "jul": "Juli",
+  "aug": "Aug",
+  "sept": "Sept",
+  "oct": "Okt",
+  "nov": "Nov",
+  "dec": "Dez",
+  "nft_purchase": "NFT Kauf",
```

Comment:
"kaufen", not "kauf"

Reply:
@heyitsjk can you confirm this because this is what it is in the files that you provided.